browser/ui/BUILD.gn

```diff
@@ -528,7 +528,16 @@ source_set("ui") {
 
     deps += [ "//brave/components/tor/resources:internals_generated_resources" ]
   }
-
+  if (brave_wallet_enabled && !is_ios) {
```

Comment:
`!is_ios` should not apply here. ios should never have a dep on `//brave/browser/ui`